plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2734,8 +2734,12 @@ elementgui.potioneffect.event_tick_condition=Active tick condition
 elementgui.potioneffect.display_name=Potion effect display name: 
 elementgui.potioneffect.category=Potion effect category: 
 elementgui.potioneffect.color=Potion effect color: 
+elementgui.potioneffect.cures=Potion cures: 
 elementgui.potioneffect.icon=Effect icon
 elementgui.potioneffect.instant=<html>Is this potion effect instantly applied?<br><small>If this is checked, potion effect tick procedure will not be called
+elementgui.potioneffect.is_cured_by_honey =Honey
```

Comment:
`oney =Honey`

I have not seen anything like this before

Reply:
![image](https://github.com/MCreator/MCreator/assets/16374228/92ce8e57-a3cc-4c3c-9bd2-ca0597fd6bf6)

Reply:
Fixed